llm_assistant/discord_bot/ai/qa.py

```diff
@@ -0,0 +1,36 @@
+import os
+import dotenv
+from openai import OpenAI, AsyncOpenAI
+
+import time
+
+dotenv.load_dotenv()
+
+
+async def async_answer_question(model, question):
+  start = time.time()
+  ai_server = os.environ.get("ai_server", "http://localhost:8000")
```

Comment:
- if `"ai_server" is an env variable, please make it ALL_CAPS
- do we have any documentations on how to fill this in? i.e. is there a change needed in the `.env` file? If yes, please also fill that in the `.env.example` file